internal/scaffold/scaffold_test.go

```diff
@@ -164,8 +166,8 @@ func Test_RunScaffold_Succeeds_WhenDefaultCRFileIsNotConfigured(t *testing.T) {
 func Test_RunScaffold_ReturnsError_WhenGeneratingDefaultCRFileFails(t *testing.T) {
```

Comment:
Same as for the test above. Needs to be updated to assert the correct error now and `defaultcr/errors.go` can be deleted.